backend/cypress/e2e/check-access.cy.js

```diff
@@ -247,13 +326,13 @@ describe('Create New Audit', () => {
         });
 
         cy.get('.auditee_contacts .delete-contact').click();
-        cy.get('.auditee_contacts .grid-row').should('have.length', 1);
+        cy.get('.auditee_contacts .grid-row').should('have.length', 3);
```

Comment:
I put the new full name fields in their own row, which means _two_ new rows are added each time you add a new contact.